### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by EvgenyShashkin**
Если не пробрасывать переменную через Dockerfile, то она не попадает в контейнер.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+language: python
+
+python:
+  - "3.5"
+
+services:
+  - docker
+  
+before_script:
+  - docker-compose -f docker-compose.yml -f docker-compose-ci.yml build
```
Comment (vdanilin) [inline .travis.yml]: Change these lines on this one:
`docker-compose -f docker-compose.yml -f docker-compose-ci.yml up --build -d `
It should resolve a problem with setting an env variable to a container.

File: Dockerfile [hunk 2]
```diff
@@ -17,6 +17,8 @@ ARG TF_ANNOTATION
 ENV TF_ANNOTATION=${TF_ANNOTATION}
 ARG DJANGO_CONFIGURATION
 ENV DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION}
+ARG COVERALLS_REPO_TOKEN
```
Comment (vdanilin) [inline Dockerfile]: There should be no changes in Dockerfile (except karma-coveralls string), remove it.

**Review by vdanilin — CHANGES_REQUESTED** [review]
Please resolve all discussions and merge the last changes from this repo to your fork repo.

File: docker-compose-ci.yml [hunk 3]
```diff
@@ -5,5 +5,6 @@ services:
     build:
       args:
         WITH_TESTS: "yes"
-        COVERALLS_REPO_TOKEN: 
+    environment:
+      COVERALLS_REPO_TOKEN: ""
```
Comment (vdanilin) [inline docker-compose-ci.yml]: delete the quotes else you get empty env variable value

**Review by vdanilin — CHANGES_REQUESTED** [review]

**Review by vdanilin — COMMENTED** [review]
@EvgenyShashkin Could you write here whether Travis CI and coveralls.io have passed successfully or not?

**Comment by EvgenyShashkin** [comment]
@DanVev Of course.

**Comment by EvgenyShashkin** [comment]
@DanVev It works.

**Comment by vdanilin** [comment]
@EvgenyShashkin I still can't merge it unless you pull latest changes to your repo and merge it to your develop branch.

**Review by vdanilin — DISMISSED** [review]

**Comment by EvgenyShashkin** [comment]
@DanVev And now?

**Comment by vdanilin** [comment]
@EvgenyShashkin could you add coveralls status badge to README.md?

**Review by vdanilin — APPROVED** [review]

**Pull request merged**